src/overlays/actors/ovl_Bg_Po_Syokudai/z_bg_po_syokudai.h

```diff
@@ -8,7 +8,11 @@ struct BgPoSyokudai;
 
 typedef struct BgPoSyokudai {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x64];
+    /* 0x014C */ u8 poeSisterColor;
+    /* 0x014E */ s16 flameTextureScroll;
```

Comment:
were you able to confirm this is actually for the flame? We shouldn't name things based on guesses, I only mentioned this being for the flame because it makes sense but I had no proof

I'll prove it now then:
`Gfx_TwoTexScroll` is used with `flameTextureScroll` before branching to dlist at `0x0404D4E0` (`gSPDisplayList(oGfxCtx->polyXlu.p++, D_0404D4E0);`)
Segment `04` is known to point to gameplay_keep data so let's investigate this offset:
<details>
<summary>
click to show gfxdis (dlist disassembly) output
</summary>

```c
$ ./gfxdis.f3dex2.exe -a 0x4D4E0 -f ../1_gameplay_keep_Global_Gameplay_Objects/zobj.zobj
{
    gsDPPipeSync(),
    gsDPSetTextureLUT(G_TT_NONE),
    gsSPTexture(qu016(0.999985), qu016(0.999985), 0, G_TX_RENDERTILE, G_ON),
    gsDPLoadTextureBlock(0x0404DE60, G_IM_FMT_I, G_IM_SIZ_8b, 32, 64, 0, G_TX_MIRROR | G_TX_CLAMP, G_TX_MIRROR | G_TX_CL
AMP, 5, 6, G_TX_NOLOD, G_TX_NOLOD),
    gsDPLoadMultiBlock_4b(0x0404D660, 0x0100, 1, G_IM_FMT_I, 32, 128, 0, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMIRROR | G_TX
_WRAP, 5, 7, G_TX_NOLOD, G_TX_NOLOD),
    gsDPSetCombineLERP(TEXEL1, PRIMITIVE, PRIM_LOD_FRAC, TEXEL0, TEXEL1, 1, PRIM_LOD_FRAC, TEXEL0, PRIMITIVE, ENVIRONMEN
T, COMBINED, ENVIRONMENT, COMBINED, 0, PRIMITIVE, 0),
    gsDPSetRenderMode(Z_CMP | IM_RD | CVG_DST_SAVE | ZMODE_XLU | FORCE_BL | GBL_c1(G_BL_CLR_IN, G_BL_0, G_BL_CLR_IN, G_B
L_1), G_RM_ZB_CLD_SURF2),
    gsSPClearGeometryMode(G_G_CULL_BACK | G_LIGHTING | G_TEXTURE_GEN | G_TEXTURE_GEN_LINEAR),
    gsSPDisplayList(0x08000000),
    gsSPVertex(0x0404D4A0, 4, 0),
    gsSP2Triangles(0, 1, 2, 0, 0, 3, 1, 0),
    gsSPEndDisplayList(),
}
```
</details>

Let's just focus on the two textures loaded. I can explain on Discord what the rest does if you're interested, ask me

```c
    gsDPLoadTextureBlock(0x0404DE60, G_IM_FMT_I, G_IM_SIZ_8b, 32, 64, 0, G_TX_MIRROR | G_TX_CLAMP, G_TX_MIRROR | G_TX_CL
AMP, 5, 6, G_TX_NOLOD, G_TX_NOLOD),
    gsDPLoadMultiBlock_4b(0x0404D660, 0x0100, 1, G_IM_FMT_I, 32, 128, 0, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMIRROR | G_TX
_WRAP, 5, 7, G_TX_NOLOD, G_TX_NOLOD),
```

The first with `0x0404DE60, G_IM_FMT_I, G_IM_SIZ_8b, 32, 64` indicates that there is a 32x64 image using format I8 (`G_IM_FMT_I, G_IM_SIZ_8b`, [texture formats](https://wiki.cloudmodding.com/oot/Textures))

Well, what would you know (look at the 32x64 box)
<details>
<summary>
click to show texture viewer screenshot
</summary>

![0x0404DE60 I8 32x64 is flame texture](https://421.es/doyu/1n8i8m)
</details>

The second `gsDPLoadMultiBlock_4b(0x0404D660, 0x0100, 1, G_IM_FMT_I, 32, 128` indicates a 32x128 I4 texture
<details>
<summary>
click to show texture viewer screenshot
</summary>

![0x0404D660 I4 32x128 is flame texture](https://421.es/doyu/1n8ie9)
</details>

So yeah it is indeed about the flame. The name is accurate :)

**BONUS**
You can see in the disassembled dlist `gsSPDisplayList(0x08000000)`, notice that this calls a dlist pointed to by segment 8, aka the dlist you setup in code with
```c
        gSPSegment(oGfxCtx->polyXlu.p++, 0x08,
                   Gfx_TwoTexScroll(globalCtx->state.gfxCtx, 0, 0, 0, 32, 64, 1, 0,
                                    (this->flameTextureScroll * -20) & 0x1FF, 32, 128));
```
:)

Reply:
While I'm not sure if it counts as proof, what I did was open it up in PJ64, lock the value, and note that the flame texture stopped.